controller/controller_server/utils.py

```diff
@@ -154,14 +153,48 @@ def validate_delete_snapshot_request(request):
     logger.debug("request validation finished.")
 
 
+def validate_validate_volume_capabilities_request(request):
+    logger.debug("validating validate_volume_capabilities volume request")
+
+    logger.debug("validating volume id")
+    volume_id = request.volume_id
+    if not request.volume_id:
+        raise ValidationException(messages.name_should_not_be_empty_message)
```

Comment:
why name?

Reply:
to align with other volume_id [vlidations](https://github.com/IBM/ibm-block-csi-driver/blob/072da7365135d458ca680c0dd6c2d4786e872b06/controller/controller_server/utils.py#L155)

Reply:
looks like a bug specifically in `validate_delete_snapshot_request`.
please do not duplicate it. instead, solve it there as well

Reply:
done

Reply:
could you please fix it in `validate_delete_snapshot_request` as well?

Reply:
thanks